src/query/service/src/interpreters/interpreter_merge_into.rs

```diff
@@ -74,8 +72,8 @@ pub struct MergeIntoInterpreter {
 }
 
 impl MergeIntoInterpreter {
-    pub fn try_create(ctx: Arc<QueryContext>, plan: MergePlan) -> Result<InterpreterPtr> {
```

Comment:
Keep Result<InterpreterPtr>

Reply:
Align with other interpreter

Reply:
`Plan::Insert` and `Plan::Replace` are not aligned.

Reply:
let's refactor them later